communicators/http_communicator.go

```diff
@@ -51,7 +63,9 @@ func (m *HTTPCommunicator) RespondWith(responses []prototype.MessagePrototype) {
 					io.WriteString(w, r.Body)
 				}
 			}
+			responseGauge.Dec()
```

Comment:
May be move this to defer? Just in case.

Reply:
fixed